banzai_floyds/tests/utils.py

```diff
@@ -66,40 +67,38 @@ def generate_fake_science_frame(include_sky=False, flat_spectrum=True, fringe=Fa
     """
     nx = 2048
     ny = 512
-    INITIAL_LINE_WIDTHS = {1: 15.6, 2: 8.6}
+    INITIAL_LINE_FWHMS = {1: 15.6, 2: 8.6}
     # DISPERSIONS = {1: 3.13, 2: 1.72}
     # Tilts in degrees measured counterclockwise (right-handed coordinates)
     INITIAL_LINE_TILTS = {1: 8., 2: 8.}
-    profile_width = 4
+    profile_fwhm = 10.0
     order_height = 93
     read_noise = 6.5
-    line_widths = [15.6, 8.6]
+    line_fwhms_angstroms = [15.6, 8.6]
     input_fringe_shift = fringe_offset
 
     order1 = Legendre((135.4, 81.8, 45.2, -11.4), domain=(0, 1700))
-    order2 = Legendre((410, 17, 63, -12), domain=(475, 1975))
+    order2 = Legendre((380, 17, 63, -12), domain=(475, 1975))
```

Comment:
I discovered the top order was falling off the image. Funny things happen when the order region does not have the same number of y pixels for each x.